quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/view/components/AbstractPawnComponent.java

```diff
@@ -129,20 +133,27 @@ public void setCurrentCoordinate(Coordinate currentCoordinate) {
     }
 
     private void drawPawn() {
-        Circle ellipse = new Circle(TILE_SIZE * 0.3215);
-        ellipse.setFill(Color.web(pawn.getType().getHexColor()));
-        ellipse.setStroke(Color.BLACK);
-        ellipse.setStrokeWidth(TILE_SIZE * 0.03);
-        ellipse.setTranslateX((TILE_SIZE - TILE_SIZE * 0.3125 * 2) / 2);
-        ellipse.setTranslateY((TILE_SIZE - TILE_SIZE * 0.26 * 2) / 2);
-        getChildren().addAll(ellipse);
+        StackPane pane = new StackPane();
```

Comment:
Use `CTRL` + `ALT` + `l` to perform automatic indentation.

Reply:
NEVER